### DISCUSSION TITLE
Suggest to use a PG version which is secured

### DISCUSSION THREAD

**Discussion by AJuliette** [rails/rails#52308] [Docs]
Right now in the doc it is suggested: "In order to use the PostgreSQL adapter you need to have at least version 9.3 installed. Older versions are not supported." in the [Active Record and PostgreSQL page](https://guides.rubyonrails.org/active_record_postgresql.html).

But PG versions under 11 are not supported anymore by the PG community (they have a 5 year maintenance policy), so we're suggesting to use PG versions which are not secured. Should we not prevent this ?

**Comment by MatheusRich** [comment]
I'd say you should open a PR to update the docs.

**Comment by flavorjones** [comment]
This part of the guide is trying to inform users of the lower limit on the version of Postgres with which the adapter is compatible. I would _hope_ that most readers can distinguish that from a suggestion to run that version, but if you'd like to suggest alternative wording that's clearer about the intention, a PR would be very welcome! ♥

More generally, though: the most recent change to the minimum postgres version supported was in 6fb128d1 by @yahonda, perhaps he knows if there's a plan to bump the min version of Postgres in an upcoming version of Rails.

**Comment by yahonda** [comment]
> perhaps he knows if there's a plan to bump the min version of Postgres in an upcoming version of Rails.

As far as I understand, there is no plan to bump the minimum versions of PostgreSQL, MySQL.
I had opened a pull request to bump the minimum version of PostgreSQL 9.5 on 2021, It did not merged https://github.com/rails/rails/pull/41526 .
